### DISCUSSION TITLE
Export getCroppedImg from package.

### DISCUSSION THREAD

**Discussion by cagosto** [ValentinH/react-easy-crop#412] [Ideas]
What do you think about exposing the `getCroppedImg` function. So others can import from `react-easy-crop`?

**Comment by ValentinH** [comment]
I don't really want to have it part of the library because this function is tricky and is often a source of issues on some browsers like Safari mobile. Therefore, I prefer only providing it as example to avoid having to actually maintain it and provide support for it.
